package.json

```diff
@@ -36,6 +36,7 @@
     "downsize": "^0.0.8",
     "esm": "^3.1.1",
     "form-data": "^2.3.3",
+    "grpc": "^1.20.3",
```

Comment:
Do we need this? Thought it was coming in as a sub dependency

Reply:
Without this, Firebase actively didn't work on my setup.